### PR TITLE
[Doc] Edit benchmark gallery in README

### PR DESCRIPTION
**Pull request by noahcao**
I remake the gallery for supported/on-the-way methods and datasets in the README.

We should discuss the aesthetic preference of this version. I use emoji ⭕ and ✅ to indicate on-the-way and already supported entries. This is a workaround because the raw html tags `<input type="checkbox" />` and `<input type="checkbox"  checked />` for checkbox in the table are not supported by Github markdown standard. There are other alternatives such as ✔️. Or, we could make the table pure HTML coded instead of a markdown table, which will nicely support in-table checkbox.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Documentation: https://mmtracking.readthedocs.io/
 
 ## Introduction
 
-MMTracking is an open source video perception toolbox by PyTorch. It is a part of [OpenMMLab](https://openmmlab.com) project.
+MMTracking is an open source video perception toolbox based on PyTorch.
+It is a part of the OpenMMLab project.
```
Comment (GT9505) [inline README.md:36]: revert the changes

File: README.md [hunk 2]
```diff
@@ -71,30 +72,51 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
...
-- [x] [STARK](configs/sot/stark) (ICCV 2021)
-
-Supported methods of video instance segmentation:
-
-- [x] [MaskTrack R-CNN](configs/vis/masktrack_rcnn) (ICCV 2019)
+<style>
+td, th {
+   border: none!important;
+}
+</style>
```
Comment (GT9505) [inline README.md:81]: Can we delete these lines?

File: README.md [hunk 3]
```diff
@@ -71,30 +72,51 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
...
+
+### Video Object Detection
+
+| Method                                                                              | Dataset                                                                  |
+| ----------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| :white_check_mark:  [DFF](configs/vid/dff) (CVPR 2017)                              | :white_check_mark: [ILSVRC](http://image-net.org/challenges/LSVRC/2017/) |
```
Comment (GT9505) [inline README.md:87]: I think we could use `[x]` and `[ ]` to indicate the supported and on the way methods. 
`white_check_mark` and `:o:` seem a little ugly.

File: README.md [hunk 4]
```diff
@@ -71,30 +72,51 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
...
+
+### Multi-Object Tracking
+
+| Method                                                                     | Dataset                                                         |
+| -------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| :white_check_mark:  [SORT/DeepSORT](configs/mot/deepsort) (ICIP 2016/2017) | :white_check_mark:  [MOT Challenge](https://motchallenge.net/)  |
+| :white_check_mark:  [Tracktor](configs/mot/tracktor) (ICCV 2019)           | :white_check_mark:    [CrowdHuman](https://www.crowdhuman.org/) |
+| :white_check_mark:  [QDTrack](configs/mot/qdtrack) (CVPR 2021)             | :white_check_mark:   [LVIS](https://www.lvisdataset.org/)       |
+| :white_check_mark:  [ByteTrack](configs/mot/bytetrack) (arXiv 2021)        | :white_check_mark:   [TAO](https://taodataset.org/)             |
+| :o: [OC-SORT](https://github.com/noahcao/OC_SORT)  (arXiv 2022)            | :o: [DanceTrack](https://dancetrack.github.io)                  |
```
Comment (GT9505) [inline README.md:111]: It is unnecessary to set the link for on the way methods

**Review by GT9505 — COMMENTED** [review]
Are you going to support CenterTrack and  mask2former in the near future?

File: README.md (hunk 3 above)
Comment (noahcao) [reply README.md:87]: I agree. But github readme does not support use [x] and [] in a table.

File: README.md (hunk 4 above)
Comment (noahcao) [reply README.md:111]: I great. Will edit them.

File: README.md (hunk 3 above)
Comment (noahcao) [reply README.md:87]: Could you provide any better solution?

File: README.md (hunk 1 above)
Comment (noahcao) [reply README.md:36]: Oh, this was a mistake. I have reverted it in the new commit.

File: README.md (hunk 2 above)
Comment (noahcao) [reply README.md]: Done. these lines are for HTML-style editing, but not supported by Github readme.

File: README.md (hunk 3 above)
Comment (GT9505) [reply README.md:80]: Maybe using the following structure?
```
### Video Object Detection

Supported Methods
- [x] aaa
- [x] aaa
- [ ] bbb

Supported Datasets
- [x] aaa
- [x] aaa
```

Comment (noahcao) [reply README.md]: Good idea. Have adjusted it in the new commit.

**Review by GT9505 — COMMENTED** [review]
1. Please review the pr by yourself from the `files changed` button, before requesting reviews. Doing this can eliminate some mistakes, such as there is no `[x]` for FGFA, SELSA and Temporal RoI Align methods.
2. We add on the way methods in the `README.md` only when a related pr has been created.

**Review by noahcao — COMMENTED** [review]
I have reviewed the latest changes.

**Review by GT9505 — APPROVED** [review]
